sci-visualization/grace/grace-5.1.25.ebuild

```diff
@@ -22,33 +22,36 @@ IUSE="fortran fftw jpeg netcdf png"
 
 DEPEND="
 	media-libs/t1lib
-	media-libs/tiff
+	media-libs/tiff:0
 	sys-libs/zlib
 	>=x11-libs/motif-2.3:0
 	x11-libs/xbae
 	fftw? ( sci-libs/fftw:2.1 )
-	jpeg? ( virtual/jpeg )
+	jpeg? ( virtual/jpeg:0 )
 	netcdf? ( sci-libs/netcdf )
-	png? ( media-libs/libpng )"
+	png? ( media-libs/libpng:0= )"
 RDEPEND="${DEPEND}
 	x11-misc/xdg-utils"
 
+PATCHES=(
+	# move tmpnam to mkstemp (adapted from debian)
+	"${FILESDIR}"/${PN}-5.1.22-mkstemp.patch
+	# fix configure instead of aclocal.m4
+	"${FILESDIR}"/${PN}-5.1.21-netcdf.patch
+	# fix for missing defines when fortran is disabled
+	"${FILESDIR}"/${PN}-5.1.21-fortran.patch
+	# fix a leak (from freebsd)
+	"${FILESDIR}"/${PN}-5.1.22-dlmodule.patch
+	"${FILESDIR}"/${PN}-5.1.22-ldflags.patch
+)
+
 pkg_setup() {
 	fortran-2_pkg_setup
-	tc-export CC
+	tc-export CC AR
```

Comment:
That was already in 5.1.23-r1, this diff compares against the older EAPI 4 build